MMTabBarView/MMTabBarView/MMMojaveTabStyle+Assets.m

```diff
@@ -77,12 +77,10 @@ - (id)assetForPart:(MMMojaveTabStyleAsset)part ofTabBarView:(MMTabBarView *)tabB
 inline static NSColor* labelColor(void)
 {
 #if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101400
```

Comment:
> Is it necessary at all to use `@import Framework` instead of `#import <Framework/Framework.h>` here? AFAIK, even when modules are used by project that uses this framework, `#import <Framework/Framework.h>` is mapped automatically to `@import Framework`.

If I use `#import <Framework/Framework.h>`, I get ugly warnings by Xcode : 
````
Treating #import as an import of module 'Framework'
````

This convulated construction is the sole solution I found to avoid warnings…

Reply:
If I understand well, this is not related to the minimum deployment target, but to the SDK used to build the framework.

As Xcode 10 and later versions both support macOS 10.13 and macOS 10.14, it seems reasonable to get rid of these checks. But I'd prefer we wait a little more, to limit divergences with upstream code.